src/components/esg/Esg.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { EsgContent, EsgImage, EsgSubtitle, EsgTitle, EsgWrapper } from './styledEsg'
+import { EsgData } from '../../constants/esgConst'
+const { title, subtitle } = EsgData;
+
+const Esg = () => {
+  return (
+    <EsgWrapper>
+      <EsgContent>
+        <EsgImage />
+        <EsgTitle>{title}</EsgTitle>
+        <EsgSubtitle>{subtitle}</EsgSubtitle>
```

Comment:
Need a highlight component here for 'the green plan' from the shared components folder so the text will be orange